src/aliases/store/postgres/postgres.go

```diff
@@ -0,0 +1,58 @@
+package aliaspg
+
+import (
+	"context"
+	goerrors "errors"
+
+	models "github.com/consensys/quorum-key-manager/src/aliases/models"
+	aliasstore "github.com/consensys/quorum-key-manager/src/aliases/store"
+	"github.com/consensys/quorum-key-manager/src/infra/postgres"
+)
+
+var _ aliasstore.Store = &Store{}
+
+// Store stores the alias data in a postgres DB.
+type Store struct {
```

Comment:
This is your `alias.go` and this is called `Alias` or `AliasRepository` or similar

Reply:
ok